src/Employee/Employee.java

```diff
@@ -0,0 +1,43 @@
+package Employee;
+
+public class Employee {
+    private double salary = 759.35;
```

Comment:
Please do not initialize here your variables with specific values because it's not universal. It's better to do it for each object separately